### PR TITLE
feat(results): styling for mobile

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

Music year: 1997

- List of changes

### Acceptance criteria

- [x]  all buttons hidden if no quizzes
- [x]  printable button is hidden on mobile
- [x]  buttons match design
- [x]  Banner is removed
- [x]  Content is viewable in mobile view
- [x]  pa11y and percy

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2839--oak-web-application.netlify.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/PupilViews/PupilResults/PupilResults.view.tsx [hunk 1]
```diff
@@ -42,19 +40,22 @@ const QuizSectionRender = (props: QuizResultsProps) => {
...
+      <OakFlex $pb={["inner-padding-xl", "inner-padding-none"]}>
+        <QuizResultInner
+          index={index}
+          questionResult={questionResult}
+          quizArray={quizQuestionArray}
+          lessonSection={lessonSection}
         />
-      )}
+      </OakFlex>
+      <OakHandDrawnHR
```
Comment (kimon-satan) [inline src/components/PupilViews/PupilResults/PupilResults.view.tsx:51]: why have you changed the method here ? this feels like a regression on the previous

File: src/components/PupilViews/PupilReview/PupilReview.view.tsx [hunk 2]
```diff
@@ -187,29 +191,32 @@ export const PupilViewsReview = (props: PupilViewsReviewProps) => {
...
-                      aria-label="Printable results, opens in a new tab"
-                      title="Printable results (opens in a new tab)"
-                      iconName={"external"}
-                      isTrailingIcon
-                      onClick={handlePrintableResultsClick}
-                      data-testid="printable-results-button"
-                    >
-                      Printable results
-                    </OakSecondaryButton>
+                    <OakBox $display={["none", "flex"]}>
```
Comment (kimon-satan) [inline src/components/PupilViews/PupilReview/PupilReview.view.tsx:194]: I don't think you need to wrap this. I think $display is also exposed on OakSecondaryButton ? (might be wrong)

**Review by kimon-satan — APPROVED** [review]
No blockers but once PRs reach this size they need better presentation. Put a testing link, screen shots and a brief description. It just makes life easier for everyone

**Comment by RBrownsett** [comment]
@benprotheroe nice-  just a couple of little changes, can the aria-label for the copy link button be updated to 'Copy link to clipboard'. It still has the old label of share results and this change will help screen readers know what action the button press takes:

image

And secondly can the padding for the 'hint used' text in mobile view be adjusted so that it does not overlap? Only happens for the last question, all other hint padding is fine

image

File: src/components/PupilViews/PupilReview/PupilReview.view.tsx (hunk 2 above)
Comment (benprotheroe) [reply src/components/PupilViews/PupilReview/PupilReview.view.tsx:194]: i tried that; didnt work

**Comment by RBrownsett** [comment]
@benprotheroe nearly there - the copyright message has turned up at the top of page as well for some reason? It only needs to be at the bottom - can this be amended? 
image

**Review by RBrownsett — APPROVED** [review]
Approved - tested on different browsers. Full testing details on Notion ticket

**Pull request merged**